mappings/net/minecraft/entity/thrown/ThrowableEntity.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_1975 net/minecraft/entity/thrown/ThrowableEntity
 	FIELD field_8421 blockX I
 	FIELD field_8422 blockY I
 	FIELD field_8423 blockZ I
+	FIELD field_8424 inBlock Lnet/minecraft/class_160;
```

Comment:
collidingWithBlock? ask @shedaniel

Reply:
I took it from the nbt

Reply:
nbt calls Worlds levels and yet we call them Worlds

Reply:
nbt is the most reliable source for entity getters, setters and field names

Reply:
its really not